index.js

```diff
@@ -603,22 +582,27 @@ function destroy() {;
 }
 
 async function lookupConversations(client, calls) {
-  let i = 0;
   const result = [];
-  const convIds = calls.map(call => call.convId);
-  if(!convIds.length){
+  const convIds = [];
+
+  calls.forEach(call => {
+    callsMap[call.convId] = call.callId;
```

Comment:
This is fine, but I think the scope of callsMap can be reduced to just the lookupConversations function, there is no need for it to be global is there?